api/swagger.yaml

```diff
@@ -0,0 +1,152 @@
+swagger: "2.0"
+info:
+  description: "Just a dictionary for **\"Галицька\"** language."
+  version: "1.0.0"
+  title: "Dictionary"
+externalDocs:
+  description: "GitHub"
+  url: "https://github.com/yushni/dictionary"
+host: "localhost"
+basePath: "/"
+tags:
+- name: "word"
+  description: ""
+schemes:
+- "http"
+paths:
+  /word:
+    post:
+      tags:
+      - "word"
+      summary: "Add a new word to the dictionary"
+      description: ""
+      operationId: "addWord"
+      consumes:
+      - "application/json"
+      produces:
+      - "application/json"
+      parameters:
+      - in: "body"
+        name: "body"
+        description: "Metadata of the word that have to be added to the dictionary"
+        required: true
+        schema:
+          $ref: "#/definitions/Word"
+      responses:
+        "200":
+          description: "Successful operation"
+          schema:
+            type: "object"
+            properties:
+              id:
+                type: "integer"
+                format: "int64"
+        "405":
+          description: "Invalid input"
+          
+  /word/{wordId}:
+    get:
+      tags:
+      - "word"
+      summary: "Get word by ID"
+      description: ""
+      operationId: "getWord"
+      produces:
+      - "application.json"
+      parameters:
+      - name: "wordId"
+        in: "path"
+        required: True
+        type: "integer"
+        format: "int64"
+      responses:
+        "200":
+          description: "successful operation"
+          schema:
+            $ref: "#/definitions/WordWithId"
+        "400":
+          description: "Invalid Word ID"
+        "404":
+          description: "Word not found"
+    
+    delete:
+      tags:
+      - "word"
+      summary: "Delete a word"
+      description: ""
+      operationId: "deleteWord"
+      parameters:
+      - name: "wordId"
+        in: "path"
+        required: True
+        type: "integer"
+        format: "int64"
+      responses:
+        "400":
+          description: "Invalid Word ID"
+        "404":
+          description: "Word not found"
+  /word/list:
+    get:
+      tags:
+      - "word"
+      summary: "Get all words"
+      parameters:
+        - name: "limit"
+          in: "query"
+          required: False
+          type: "integer"
+          format: "int64"
+          default: 20
+        - name: "offset"
+          in: "query"
+          required: False
+          type: "integer"
+          format: "int64"
+          default: 1
+      responses:
+        "200":
+          description: "Successful operation"
+          schema:
+            type: "array"
+            items:
+              $ref: "#/definitions/WordWithId"
+          
+definitions:
+  Word:
+    type: "object"
```

Comment:
all of this params are required add a **required tag** for all of them 
https://stackoverflow.com/a/51830005